packages/e2e-tests/tests/composer.spec.ts

```diff
@@ -56,9 +58,28 @@ test.beforeAll(async ({ browser, isChatmail }) => {
   chatList = page.getByLabel('Chats').getByRole('tablist')
   textarea = page.locator('textarea#composer-textarea')
   composerSection = page.getByRole('region', { name: 'Write a message' })
+  composerReply = composerSection.getByRole('region', { name: 'Reply' })
   await createNDummyChats(page, 3, 'Some chat ')
 })
 
+function getRemoveQuoteOrFileButton(parent: Locator): Locator {
+  return parent.getByRole('button', {
+    name: /(close|cancel|remove|delete)/i,
+  })
+}
+async function testDraftIsEmpty() {
+  await expect(textarea).toBeEmpty({ timeout: 500 })
+
+  await expect(composerSection).toHaveText('', { timeout: 500 })
+  // The above check should be enough, but let's double check
+  // with these extra ones.
+  await expect(
+    composerSection.getByRole('region', { name: /reply|attachment|/i })
```

Comment:
when would this be visible?

Reply:
It's the "quote reply" section:

<img width="299" height="253" alt="image" src="https://github.com/user-attachments/assets/dd7b24ba-f212-449f-8016-d11e023e6a9e" />